### PR TITLE
Adding contributing guide

### PR DESCRIPTION
**Pull request by williamcaban**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [X] Documentation Update

## Description

Adding CONTRIBUTING.md

### CODE REVIEW

**Review by thoraxe — CHANGES_REQUESTED** [review]
can you update readme.md to point to this guide?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# Upgrade pip to the most recent version
+pip install --upgrade pip
+
+# Install dependencies
+pip install -r requirements.txt
+```
+
+Happy hacking!
+
+## Before a PR
```
Comment (bparees) [inline CONTRIBUTING.md:36]:
```suggestion
## Updating Dependencies
```

**Review by bparees — COMMENTED** [review]
one nit and lgtm.

**Comment by williamcaban** [comment]
> can you update readme.md to point to this guide?

Done.

File: CONTRIBUTING.md (hunk 1 above)
Comment (williamcaban) [reply CONTRIBUTING.md]: done

**Comment by bparees** [comment]
hrm.  Thought /lgtm from an approver got the approve label automatically but apparently not in our configuration.

/approve

**Pull request merged**